apps/web/components/shared/kyc/kyc-5/types.ts

```
@@ -0,0 +1,30 @@
+	onStepChange: (step: number) => void
+	kycData: {
+		personalInfo: {
+			fullName: string
+			dateOfBirth: string
+			nationality: string
+		}
+		documents: {
+			documentType: string
+			frontImage: File | null
+			backImage: File | null
+			extractedData: any
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)